split-webapp/split/src/pages/Home.js

```diff
@@ -2,23 +2,25 @@ import React from "react";
 import { Route, NavLink, HashRouter } from "react-router-dom";
 import Transactions from "./Transactions";
 import Split from "./Split";
-import "../App.css";
 import NavList from "../components/NavList";
 import MenuBar from "../components/MenuBar";
+import styles from "./Home.module.css";
+
+console.log(styles);
```

Comment:
Is this necessary since we can just look at the styles in the Home.module.css directly? It might help removing it to avoid clutter on the terminal when running the code.

Reply:
Agreed, removed.